src/docs/resources/faq.md

```diff
@@ -364,11 +364,19 @@ and there is approximately 450.4&nbsp;KB of (compressed) ICU data.
 These numbers were measured using [apkanalyzer](https://developer.android.com/studio/command-line/apkanalyzer),
 which is also built into [Android Studio](https://developer.android.com/studio/build/apk-analyzer).
 
+On iOS, a release IPA of the same app has a download size of 10.8&nbsp;MB on an
+iPhone X as reported by Apple's App Store Connect. The IPA is larger than the
+APK mainly because Apple encrypts binaries within the IPA making the compression
+less efficient (see "iOS App Store Specific Considerations" in
+[Apple's documentation](https://developer.apple.com/library/archive/qa/qa1795/_index.html).
+
 Of course, YMMV, and we recommend that you measure your own app,
-by running `flutter build apk` and loading
+by running `flutter build apk` and loading the APK
 `build/app/outputs/apk/release/app-release.apk` into Android Studio
-([instructions](https://developer.android.com/studio/build/apk-analyzer)) for a
-detailed size report.
+([instructions](https://developer.android.com/studio/build/apk-analyzer)) or by
```

Comment:
This is a long sentence that's not unparallel. Let's split it up. Maybe:

Of course, YMMV, and we recommend that you measure your own app. To measure an Android app, run `flutter build apk` and load the APK (`build/app/outputs/apk/release/app-release.apk`) into Android Studio ([instructions](https://developer.android.com/studio/build/apk-analyzer)) for a detailed size report. To measure an iOS app, upload a release IPA to Apple's App Store Connect ([instructions](https://flutter.io/docs/deployment/ios)) to get a detailed size report.